include/glow/Importer/CommonOperatorLoader.h

```diff
@@ -135,6 +152,24 @@ class CommonOperatorLoader : public ProtobufLoader {
     nodeValueByName_[op.output(0)] = NodeValue(N, 0);
   }
 
+  void loadMinMax(llvm::StringRef typeName, const OpType &op,
+                  ArgumentDictionaryTy &dict) {
+    const std::string &opName = loadOperatorName(op);
+    auto in0 = getNodeValueOrCreateVariableByName(op.input(0));
```

Comment:
nit: lhs and rhs for inputs.

Reply:
I simply reused the naming I found in other functions. For instance `loadArithmetic` uses `in0` and 'in1'. I didn't see any usage of `lhs` and `rhs` in this loader file.

Reply:
Ok, I'll just merge this as is, and if we want to make them all consistent we can do so in a future PR.